srml/council/src/seats.rs

```diff
@@ -501,10 +509,11 @@ impl<T: Trait> Module<T> {
 		}
 	}
 
-	/// Finalize the vote, removing each of the `removals` and inserting `seats` of the most approved
-	/// candidates in their place. If the total council members is less than the desired membership
+	/// Finalize the vote, removing each of the expiring member seats and inserting seats of the most approved
+	/// candidates in their place. If the total number of council members is less than the desired membership,
 	/// a new vote is started.
-	/// Clears all presented candidates, returning the bond of the elected ones.
+	///
```

Comment:
See https://github.com/paritytech/substrate/pull/2447/files#r288417622 for replacement
```suggestion
```